lib/plugins/aws/package/compile/events/apiGateway/lib/hack/updateStage.test.js

```diff
@@ -244,4 +269,17 @@ describe('#updateStage()', () => {
       expect(context.apiGatewayRestApiId).to.equal('someRestApiId');
     });
   });
+
+  it(
+    'should not apply hack when restAPIId could not be resolved and ' +
+      'no custom settings are applied',
+    () => {
+      context.serverless.service.provider.apiGateway = {
+        restApiId: { 'Fn::ImportValue': 'RestApiId-${self:custom.stage}' },
+      };
+      return updateStage.call(context).then(() => {
+        expect(providerRequestStub.callCount).to.equal(0);
```

Comment:
Should we also spy on the log and see if smth. is logged to ensure that the user knows what's goind on?

Reply:
In this case intention was to not release any warning, just succeed.

We see there's no customization to those setting sintended, so it seems safe to just exit early without a word.

When user will attempt to customize it'll receive an informative error.

Or do you feel, we should go a bit differently?

Reply:
Makes sense. Let's keep it simple :+1: